common/djangoapps/django_sudo/utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+sudo.utils
+"""
+import django.contrib.sessions.middleware
+import sudo.middleware
+
+
+def region_name(region):
```

Comment:
I'd still like an answer to this. It doesn't seem like the act of making the region name safe to store in a cookie is edx specific code.